### ISSUE TITLE
Let plugin plug there configuration into `kinto init` command

### ISSUE THREAD

**Issue by Natim** [Kinto/kinto#721]
Most plugins have specific configuration.
We could let them hook in the `kinto init` command when installed in the venv to generate their default configuration in this file and help the user to change the mandatory ones.

**Comment by leplatrem** [comment]
> when installed in the venv

Is that feasible?

**Comment by Natim** [comment]
> Is that feasible?

Yes it is with setup.py entry_points: http://stackoverflow.com/a/9615473/186202
